agent/statemanager/testdata/v27/environmentFiles/ecs_agent_data.json

```diff
@@ -0,0 +1,168 @@
+{
+  "Data": {
+  "Cluster": "state-file",
+  "ContainerInstanceArn": "arn:aws:ecs:us-west-2:984736093387:container-instance/ea27e41b-c6e4-45a9-a7a0-484c95abece7",
```

Comment:
maybe change account id/task id/instance id to dummy value

Reply:
Fixed by replacing with dummy id and garbling the sha etc.